src/incubator/TextField/ValidationMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useContext} from 'react';
+import {TextStyle, StyleSheet} from 'react-native';
+import Text from '../../components/text';
+import FieldContext from './FieldContext';
+
+export interface ValidationMessageProps {
+  enableErrors?: boolean;
+  validationMessage?: string;
+  validationMessageStyle?: TextStyle;
+  retainSpace?: boolean;
```

Comment:
This how it started and I ended up with this solution. 
If i'll move the logic out, I will also need to move out the check for `context.isValid` (cause they go together). 
And the whole idea was to keep the logic of each internal component inside and remove complexity from the main TextField component. 

It's not that it's not possible, but It's cleaner way. 

Also, Why `enableErrors` is redundant? 
That's the only way for the user to decided whether to show the validation message or not.

Reply:
The 'enableErrors' prop issue is the same as we discussed before with TextField props that affect the render of the subcomponent. Since you're taking a different approach it doesn't matter anymore. I'll resolve this comment as well.